ofrak_type/setup.py

```diff
@@ -32,7 +32,7 @@ def run(self):
             "fun-coverage==0.2.0",
             "hypothesis~=6.39.3",
             "mypy==0.942",
-            "pytest",
+            "pytest<8.0",
```

Comment:
Do we have an understanding of why this is needed? It's not intuitive why limiting a version is required when bumping the Python version.

Reply:
@whyitfor I put it in the log message for that commit - this is needed because otherwise we get 8.0.0 by default and hit https://github.com/pytest-dev/pytest/issues/11890
https://github.com/TvoroG/pytest-lazy-fixture/issues/65, and I did not take time to investigate whether there are any better workarounds. This is not needed with python3.7 because starting with version 8.0.0 (just released Jan 27!), pytest requires python 3.8.

Reply:
Great, and now we have that record in the PR as well.